README.md

```diff
@@ -0,0 +1,66 @@
+# maxwells-daemon
+
+[![Build Status](https://img.shields.io/travis/thumbtack/maxwells-daemon.svg)(https://travis-ci.org/thumbtack/maxwells-daemon)
+[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/aws/aws-sdk-go/blog/master/LICENSE.txt)
```

Comment:
Doesn't the X11 license have an extra advertising clause not in the license we use?  I lifted our license from `ttinjector`.  Or do we use BSD3 now?